src/components/CategoryList.js

```diff
@@ -0,0 +1,76 @@
+import { useState, useEffect } from 'react';
+
+// eslint-disable-next-line import/order
+import { getFolderLink } from '../api/BaseUrl';
+
+import '../styles/CategoryList.css';
+
+import iconDelete from '../assets/logo/icon-delete.svg';
+import iconPen from '../assets/logo/icon-pen.svg';
+import iconShare from '../assets/logo/icon-share.svg';
+import iconPlus from '../assets/logo/plus.svg';
+
+import CardList from './CardList';
+import Category from './Category';
+
+const CategoryList = ({ folderInfo, categoryList, isIconVisible }) => {
+  const [selected, setSelected] = useState({ id: '', name: '전체' });
+  const [folderLink, setFolderLink] = useState(null);
+
+  const categoryData = categoryList?.data;
+
+  const handleLoad = async categoryId => {
+    try {
+      const state = await getFolderLink(categoryId);
+      setFolderLink(state);
+    } catch (error) {
+      console.error(error);
+    }

```

Comment:
P2:
getFolderLink의 getRequest 함수에서 에러처리를 해주는데 여기도 필요할까요?

```suggestion
    try {
      const state = await getFolderLink(categoryId);
      setFolderLink(state);
    } catch (error) {
      console.error(error);
    }

```